src/Controls/samples/Controls.Sample/MauiProgram.cs

```diff
@@ -119,6 +121,14 @@ public static MauiApp CreateMauiApp()
 #endif
 			});
 
+			services.AddSingleton<IImageSourceHttpClientFactory, ImageSourceHttpClientFactory>();
+			services
+				.AddHttpClient(ImageSourceHttpClientFactory.DefaultHttpClientName, httpClient =>
+				{
+					httpClient.Timeout = TimeSpan.FromSeconds(5);
+					httpClient.DefaultRequestVersion = HttpVersion.Version20;
```

Comment:
Default timeout is [100 seconds](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.timeout?view=net-7.0#remarks). Why do you change this to 5 seconds? That is an incredibly reduced amount.

Default requested version is [1.1](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.defaultrequestversion?view=net-7.0#remarks), why do you change that to 2.0?

Reply:
The sample application has no need to tweak the HttpClient at all. So those are just arbitrary values to show that configuration is possible. I would be happy to change this section. Or even remove the whole thing from the sample application.

Reply:
I would probably remove these options and put a comment like `// Optionally configure the HttpClient here`.

Reply:
Or maybe show adding a default header like "user agent" to be the app's name (I think we've seen people actually ask for that).

Reply:
Super. I'll take that into account. Let's discuss the issue for some days and I'll do code changes.